app/middlewares/user.js

```diff
@@ -0,0 +1,10 @@
+const { validateUserData } = require('../utils/user'),
```

Comment:
Utils should have only generic modules (like time,arrays, etc), business logic or specific logic should move out of here, for this case we can create validations files  (one per entity or only one)